ETL/Provision/test_provision_lambda.py

```diff
@@ -0,0 +1,58 @@
+# pylint: skip-file
```

Comment:
remove this line and add docstring to explain document

Reply:
Good idea thanks!